### PR TITLE
BOM-2913: Replace Travis with GitHub CI

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-2913](https://openedx.atlassian.net/browse/BOM-2913)

### Description
- Replaced `Travis` with `GitHub CI`.

### CODE REVIEW

**Comment by UsamaSadiq** [comment]
@kashifch could you please review the PR and remove the required Travis check from the repo settings? Thanks.

**Comment by natabene** [comment]
@georgebabey Please review and merge before 11/16.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+      - cdodge/mcka-master
```
Comment (georgebabey) [inline .github/workflows/ci.yml:7]: We can remove these `cdodge` branches

**Review by georgebabey — APPROVED** [review]
Just one minor comment/cleanup. Otherwise looks great, thank you

**Comment by UsamaSadiq** [comment]
@georgebabey  can you please also remove the required Travis check from the repo settings?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (UsamaSadiq) [reply .github/workflows/ci.yml]: Removed the extra branches.

**Comment by natabene** [comment]
@georgebabey When you have a chance, can you look into this agian?

**Pull request merged**